style.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #fff;
+  width: 375px;
+  height: 718px;
```

Comment:
- [ ] For this section is a requirement to style your components using flex-box. You can achieve this template style using flex-box, grid, padding, and margin. Please avoid using fixed values for width and height